numba/cuda/cudadrv/driver.py

```diff
@@ -458,6 +413,81 @@ def met_requirement_for_device(device):
                                (device, MIN_REQUIRED_CC))
 
 
+class _SizeNotSet(object):
+    """
+    Dummy object for _PendingDeallocs when *size* is not set.
+    """
+    def __str__(self):
+        return '?'
+
+    def __int__(self):
+        return 0
+
+_SizeNotSet = _SizeNotSet()
+
+
+class _PendingDeallocs(object):
+    """
+    Pending deallocations of a context (or device since we are using the primary
+    context).
+    """
+    MAX_PENDING_DEALLOCS_COUNT = config.CUDA_DEALLOCS_COUNT
```

Comment:
Just a nit, but perhaps it would be better to always read those from `config`, in case someone wants to change them dynamically?